pkg/services/user/identity.go

```diff
@@ -46,11 +46,13 @@ func (u *SignedInUser) NameOrFallback() string {
 // TODO: There's a need to remove this struct since it creates a circular dependency
 
 // DEPRECATED: This function uses `UserDisplayDTO` model which we want to remove
+// this function uses dto to getch the gravatar url
 func (u *SignedInUser) ToUserDisplayDTO() *UserDisplayDTO {
 	return &UserDisplayDTO{
 		ID:    u.UserID,
 		Login: u.Login,
 		Name:  u.Name,
+		// AvatarURL: dtos.GetGravatarUrl(u.GetEmail()),
```

Comment:
commented code doesn't match the description?

Reply:
Added a more verbose description. Please let me know if this works for you